### ISSUE TITLE
mkvtoolnix-gui command line

### ISSUE THREAD

**Issue by AlanHK** [mbunkus/mkvtoolnix#1251]
I'm used to using the old merge GUI, loading a file from a command shell (Far) in WinXP using
`mmg file.mkv`
then setting whatever options in the gui.

I just upgraded MKVToolnix to 8.0.
I can't get mkvtoolnix-gui to load any files from the command line.
I see in the manual:
> mkvtoolnix-gui [[preferences-file.mtxcfg] | [source-file.ext]] 

But when I do this:
`mkvtoolnix-gui file.mkv` 
I just get the gui with no files loaded, and I have to find and select the files from the gui.
Is this a bug or user error?

When I wanted to just go back to mmg, I find that it now nags me every time I start it to upgrade to mkvtoolnix-gui. Can this be made an option I can turn off after seeing it once?

**Comment by mbunkus** [comment]
Thanks for your feedback. You're not doing anything wrong. The command-line handling hasn't been implemented yet. It is on my TODO list for the next release. As there's an issue for that already (#1209) I'm going to close this one as a duplicate.

About mmg: I will switch mmg to display that warning only once for the next release. You can use the mmg from 7.9.0 until then if you want to get rid of the warning right away.

**Comment by AlanHK** [comment]
Thanks.
I tried to use mmg from 7.9, but it fails:
>Procedure entry point RegGetValueA could not be located in ADVAPI32.dll"
Looks like I will have to roll back the whole thing.

**Comment by AlanHK** [comment]
PPS -- problem is with 7.9 in general. If I install it on my WinXP nothing works, only get 
>Procedure entry point RegGetValueA could not be located in ADVAPI32.dll

I went back to 7.8 (which is what I had before I upgraded to 8.0), and 7.8 works.
The mmg from 7.8 works with 8.0 backend, but it also has a nag "incompatible mkvmerge" every time I run it. 

So I uninstalled 8.0 completely and went back to 7.8.
I will wait for 8.1 and give that a try.
